### PR TITLE
Feature/search engine 

### PR DESCRIPTION
**Pull request by JuanAyulo**

### CODE REVIEW

File: devtools_options.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+extensions:
```
Comment (cooper-73) [inline devtools_options.yaml:1]: Esta linea no va

File: lib/common/widgets/continuous_qualification_object.dart [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  @override
+  Widget build(BuildContext context) {
+    int index = value.floor();
+    return SizedBox(
+      height: 24,
+      width: width * 0.99,
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: [
+          ElevatedButton(
```
Comment (cooper-73) [inline lib/common/widgets/continuous_qualification_object.dart:64]: Use Chips widget

File: lib/common/widgets/continuous_qualification_object.dart [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+            style: ButtonStyle(
+              minimumSize: MaterialStateProperty.all(const Size(62, 24)),
+              backgroundColor: MaterialStateProperty.all(color),
+            ),
+            child: Text(
+              selectedRating[index],
+              maxLines: 1,
+              style: const TextStyle(
+                fontSize: 9,
+                fontFamily: 'Inter',
```
Comment (cooper-73) [inline lib/common/widgets/continuous_qualification_object.dart:75]: We don't use this font family in the project

**Review by cooper-73 — CHANGES_REQUESTED** [review]

File: lib/features/teachers/presentation/widgets/optional_qualifications.dart [hunk 4]
```diff
@@ -26,22 +26,25 @@ class OptionalQualifications extends StatelessWidget {
   Widget build(BuildContext context) {
     return Column(
       children: [
-        ContinuousBarQualification(
+        OptionalQualificationTag(
           asset: 'clip',
           text: 'Carga Académica',
           color: AppTheme.secondary1Color,
+          selectedRating: const ['No sé', 'Nunca', 'A veces', 'Siempre'],
```
Comment (cooper-73) [inline lib/features/teachers/presentation/widgets/optional_qualifications.dart:33]: Replace "No sé" with "-"

Comment (cooper-73) [inline lib/features/teachers/presentation/widgets/optional_qualifications.dart:33]: Replace 'No se' with '-'

File: lib/features/teachers/presentation/widgets/optional_qualifications.dart [hunk 5]
```diff
@@ -26,22 +26,25 @@ class OptionalQualifications extends StatelessWidget {
...
           value: getQualificationValueByCode(4),
         ),
-        ContinuousBarQualification(
+        OptionalQualificationTag(
           asset: 'clock',
           text: 'Exigencia',
-          color: AppTheme.secondary2Color,
+          color: AppTheme.secondary1Color,
+          selectedRating: const ['No sé', 'Nunca', 'A veces', 'Siempre'],
```
Comment (cooper-73) [inline lib/features/teachers/presentation/widgets/optional_qualifications.dart:40]: Replace "No sé" with "-"

File: lib/features/teachers/presentation/widgets/optional_qualifications.dart [hunk 6]
```diff
@@ -26,22 +26,25 @@ class OptionalQualifications extends StatelessWidget {
...
           value: getQualificationValueByCode(5),
         ),
-        ContinuousBarQualification(
+        OptionalQualificationTag(
           asset: 'pencil',
           text: 'Toma de asistencia',
-          color: AppTheme.secondary3Color,
+          color: AppTheme.secondary1Color,
+          selectedRating: const ['No sé', 'Nunca', 'A veces', 'Siempre'],
```
Comment (cooper-73) [inline lib/features/teachers/presentation/widgets/optional_qualifications.dart:47]: Replace "No sé" with "-"

**Review by cooper-73 — CHANGES_REQUESTED** [review]

File: lib/common/widgets/continuous_qualification_object.dart (hunk 3 above)
Comment (JuanAyulo) [reply lib/common/widgets/continuous_qualification_object.dart:75]: This font was used on teacher_rating_step 2 in the continous calification part (request by Figma), so as I recycle the same object with new parameters, the style remains the same. If change is needed, please tell me wich font family I must use instead.
image

Comment (cooper-73) [reply lib/common/widgets/continuous_qualification_object.dart]: Figma seems to be wrong. Default font family should be used. Modification was done.

**Review by cooper-73 — APPROVED** [review]

**Pull request merged**